Dockerfile

```diff
@@ -1,20 +1,15 @@
-# --- Build the plugin cli first ---
-FROM golang:1.18-alpine as builder
+FROM golang:1.21 as build
 
-ENV GO111MODULE=on
-WORKDIR /app
-
-COPY go.mod .
-COPY go.sum .
+WORKDIR /go/src/app
+COPY go.mod go.sum ./
 RUN go mod download
 COPY . .
-
-RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -o /go/bin/drone-helm ./cmd/drone-helm
+RUN CGO_ENABLED=0 go build -o /go/bin/app ./cmd/drone-helm
 
 # --- Copy the cli to an image with helm already installed ---
-FROM alpine/helm:3.8.1
+FROM alpine/helm:3.13.3
```

Comment:
I do not think we should move the helm version in this change.

Reply:
any particular reason?

Reply:
Limits the scope of change and what needs to be tested.  If we are upgrading go.mod and helm, we need to test and evaluate that. 

If this PR is just focused on making this image available in arm64, then what we need to test is smaller

Reply:
Just note, I am currently going to be reading all the release notes between 3.8.1 and 3.13.3

Reply:
Reverted the change to use 3.8.1